antora.yml

```diff
@@ -1,5 +1,5 @@
 name: studio
 title: Studio
-version: '7.x'
+version: '7.16'
```

Comment:
You're changing this for the archive site, right?

Reply:
Yes. Thanks!